mender/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## Version 5.2.7
 * Traefik updated to `v2.10.5`
+* Split single db-migration job into multiple jobs
```

Comment:
I'm not 100% sure where this change falls under the semantic versioning, but I believe it at least require a minor version increment since it changes the values interface.

Reply:
Let's bump to 5.3.0, even there's no breaking change.
I think we could wait for the new Mender 3.6 fix for publishing it